### ISSUE TITLE
Thing to try :)

### ISSUE THREAD

**Issue by ghost** [DidUknowiPwn/T4M#4]
You should try and get live compile on the gsc files. You save. It loads. In game. I have it working on my patch. Enjoy. Try and get it working its fucking useful.

**Comment by iAmThatMichael** [comment]
fs_basegame you mean?
You could load external files through that and such.

**Comment by ghost** [comment]
I mean when you are working with a mod you have to restart the level to update the code. I was asking you to try and make that bit live. So while ingame if a code change is made it takes instant affect.

**Comment by n123q45** [comment]
no bone limit

**Comment by iAmThatMichael** [comment]
Yeah you could do that with fs_usedevdir and then calling a dev/map/_restart.

For bone limit that's something I'll take a look at.

**Comment by n123q45** [comment]
also while your looking into the bone limit... many people people want to be able to import bigger models, which would mean to remove the vert tris limit for them. I know this isnt part of the dll since its in converter.exe, but a friend wanted me to ask you. (I rarely have this issue tho =__= cough cough http://i.snag.gy/fwjWF.jpg cough cough)

**Comment by iAmThatMichael** [comment]
Bone limit is impossible since it's tied into way more than just a simple value change, degenerate tris is related to converter not to the game itself. Finding where that is removed is more a problem than you think :|
